www/js/containers/login-page.js

```diff
@@ -1,51 +1,115 @@
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  login: {
+    marginTop: 20,
+    marginBottom: 20,
+    width: '100%'
+  },
+  options: {
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    color: '#B3B3B3'
+  }
+};
+
+class LoginPage extends React.Component {
+  constructor( props ) {
+    super( props );
+    bindAll( this, 'onLogin' );
+  }
 
   componentDidMount() {
-    const { setDrawer } = this.props;
-    setDrawer('Login');
+    const {setDrawer} = this.props;
+    setDrawer( 'Login' );
+  }
+
+  onLogin() {
+    const email = this.refs.email.getValue();
+    const password = this.refs.password.getValue();
+
+    this.props.dispatch( login( email, password ) ).then( ( ) => {
+      this.props.history.push( '/' );
+    } );
   }
 
   render() {
-    const imgStyle = {
-      maxHeight: '150px',
-      display: 'block',
-      margin: 'auto'
-    }
-    const centerText = {
-      textAlign: 'center'
-    }
     return (
-      <div className="form-column page-content">
-        <img style={imgStyle} src="./img/advc.png"/>
-        <div>
-          <div className="form-row">
-            <TextField floatingLabelText="Email..."/>
-          </div>
-          <div className="form-row">
-            <TextField type="password" floatingLabelText="Password"/>
-          </div>
-          <div className="form-row">
-            <RaisedButton secondary> Log In </RaisedButton>
-          </div>
-          <div className="form-row">
-            <div style={centerText}>
-              <CardText>
-                <Link to="register">Sign Up</Link> for Adventure Cycling
-              </CardText>
-              <CardText>
-                <a>Forgot Password?</a>
-              </CardText>
+      <div style={ styles.border }>
+        <div style={ styles.page }>
+          <img style={ styles.logo } src="./img/advc-big.jpg" />
+          <Paper style={ styles.block } zDepth={ 2 }>
+            <TextField hintText="Email"
+              underlineShow={ true }
+              fullWidth={ true }
+              ref="email" />
+            <TextField hintText="Password"
+              underlineShow={ true }
+              fullWidth={ true }
+              ref="password"
+              type="password" />
+            <RaisedButton style={ styles.login }
+              secondary
+              onClick={ this.onLogin }>
+              Log In
+            </RaisedButton>
+            <div style={ styles.options }>
+              <p>
+                <Link to="register"> Sign Up
+                </Link> for Adventure Cycling
+              </p>
+              <p>
+                Forgot password?
+              </p>
             </div>
-          </div>
+          </Paper>
         </div>
       </div>
-    );
+      );
   }
 }
 
-export default LoginPage;
+function select( state ) {
+  return {
+    account: state.account
```

Comment:
you can just do a `connect()(LoginPage)`, and if we're not using account, let's just do that for now.